### PR TITLE
Add kvm 11.3.0 with flatcar

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Comment by mazzy89** [comment]
Is this a duplicate of this https://github.com/giantswarm/releases/pull/125?

**Comment by mazzy89** [comment]
Is this a duplicate of this https://github.com/giantswarm/releases/pull/125?

**Comment by MarcelMue** [comment]
> Is this a duplicate of this #125?

Yes I didn't see #125 - I will keep this one as it is already updated and has no conflicts with master.

**Comment by mazzy89** [comment]
No problem. going to close than that one.

File: release-notes/kvm/v11.3.0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## :zap: Giant Swarm Release 11.3.0 for KVM :zap:
+
+This release includes Kubernetes v1.16.9 as well as some reliability and user experience improvements.
```
Comment (MartaD) [inline release-notes/kvm/v11.3.0.md:3]:
```suggestion
This release includes Kubernetes v1.16.9 as well as reliability and user experience improvements.
```

**Comment by MarcelMue** [comment]
Everything is merged and prepared. Going to merge here and announce release tomorrow morning.

File: kvm.yaml [hunk 2]
```diff
@@ -3,9 +3,60 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v11.2.3
+  name: v11.3.0
 spec:
   state: active
+  date: 2020-04-24T12:00:00Z
```
Comment (tfussell) [inline kvm.yaml:9]:
```suggestion
  date: 2020-04-27T12:00:00Z
```

File: release-notes/kvm/v11.3.0.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Below, you can find more details on components that were changed with this release.
+
+### Kubernetes [v1.16.9](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v1168)
+- Updated from v1.16.3.
+
+### kvm-operator [v3.11.0](https://github.com/giantswarm/kvm-operator/releases/tag/v3.11.0)
+- Use Flatcar linux instead of CoreOS.
+- Streamlined image templating for core components for quicker and easier releases in the future.
+- Support setting OIDC username and groups prefix.
+- Enabled per-cluster configuration of kube-proxy's `conntrackMaxPerCore` parameter.
```
Comment (tfussell) [inline release-notes/kvm/v11.3.0.md:19]: Should we have a changelog for Flatcar? The Brazil timezone?

Comment (MarcelMue) [reply release-notes/kvm/v11.3.0.md:19]: Good point - I would add the flatcar changelog from `9.0.1`:
https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v9.0.1.md#flatcar-linux-234531

No brazil timezone I would say :D

Comment (tfussell) [reply release-notes/kvm/v11.3.0.md:19]: Makes sense. From a customer perspective I might be interested in knowing the new kernel version (4.19.68->4.19.107) and OpenSSL, but the rest isn't very relevant.

**Comment by MarcelMue** [comment]
All done going with this.

**Pull request merged**